tests/test_system_eraser.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+import json
+from unittest.mock import MagicMock, patch
+
+from ros.processor.system_eraser import SystemEraser
+
+
+@pytest.fixture
+def mock_app_context():
+    with patch("ros.lib.app.app.app_context") as mock_ctx:
+        yield mock_ctx
+
+
+@pytest.fixture
+def mock_db_session():
+    with patch("ros.extensions.db.session") as mock_db:
+        mock_db.session.execute = MagicMock()
+        mock_db.session.commit = MagicMock()
+        yield mock_db
+
+
+@pytest.fixture
+def mock_consumer():
+    with patch("ros.processor.system_eraser.consume.init_consumer") as mock_consumer_init:
+        consumer = MagicMock()
+        mock_consumer_init.return_value = consumer
+        yield consumer
+
+
+def test_delete_system_success(mock_app_context, mock_db_session, mock_consumer):
+    eraser = SystemEraser()
+
+    mock_query_object = MagicMock()
+    with patch("ros.extensions.db.delete", return_value=mock_query_object):
```

Comment:
Rest change looks good. Could you add new test or modify this one where by creating test system, it simply runs delete_system method and returns expected result to verify?